src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -299,33 +300,4 @@ export default {
 	border-bottom: 1px solid var(--color-border-dark);
 }
 
-.navigation {
```

Comment:
I strongly suggest you do a tour of your code to check where are the other locations you did something similar and nuke it :wink:

Reply:
lol, ok :)